src/types/instantsearch.ts

```diff
@@ -12,23 +12,37 @@ export type SearchParameters = any;
 
 export type SearchResults = any;
 
+type HitAttributeHighlightResult = {
+  value: string;
+  matchLevel: 'none' | 'partial' | 'full';
+  matchedWords: string[];
+  fullyHighlighted?: boolean;
```

Comment:
Is it really optional?

Reply:
When it's not fully highlighted, the key is not returned:

![image](https://user-images.githubusercontent.com/6137112/55795602-856d4280-5ac8-11e9-8bce-75bbb005819f.png)

When it is fully highlighted, the attribute is `true`:

![image](https://user-images.githubusercontent.com/6137112/55795642-94ec8b80-5ac8-11e9-9434-2a7331cbe113.png)

Reply:
wouldn't in that case the type be `fullyHighlighted?: true`? Although I guess technically it could still be false, and best to leave that case open